l10n_nl_tax_statement/README.rst

```diff
@@ -0,0 +1,186 @@
+=========================
+Netherlands BTW Statement
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--netherlands-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-netherlands/tree/12.0/l10n_nl_tax_statement
+    :alt: OCA/l10n-netherlands
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-netherlands-12-0/l10n-netherlands-12-0-l10n_nl_tax_statement
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/176/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides you with the Tax Statement in the Dutch format.
+
+Use this module in combination with module ``l10n_nl_tax_invoice_basis`` if you want to adopt the *Factuurstelsel* system for your Company.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+* This module depends on module ``account_tax_balance`` available at https://github.com/OCA/account-financial-reporting.
```

Comment:
Module doesn't depend on account_tax_balance anymore?

Reply:
Not anymore, because with Odoo 13 the tax tags and other needed tax information are available in move lines directly.
Was used in previous versions because the tax information was in account.tax only and module `account_tax_balance` was an help to retrieve the needed move lines.